configs/defaults/general.yaml

```
@@ -4,3 +4,5 @@ intermediate_movements:
 - "config"
 - "bin"
 - "input"
+compute_file_checksums: True
+parallel_file_movements: threads
```

For the time being let's keep this both false. Once dask parallelization is available we can turn them on by default.
```suggestion
compute_file_checksums: False
parallel_file_movements: False
```